javascript/classes.json

```diff
@@ -489,7 +489,8 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": "81",
+              "version_added": false,
+              "notes": "Available only in nightly builds. See <a href='https://bugzilla.mozilla.org/show_bug.cgi?id=1562054'>bug 1562054</a>.",
```

Comment:
These links need shortening, which is why the linter is failing. See https://github.com/mdn/browser-compat-data/pull/8682/checks?check_run_id=1668964873#step:5:2339